llvm/lib/TargetParser/Host.cpp

```diff
@@ -1151,6 +1151,25 @@ static const char *getAMDProcessorTypeAndSubtype(unsigned Family,
       break; //  "znver4"
     }
     break; // family 19h
+  case 26:
```

Comment:
Can you bump the equivalent code in `compiler-rt` too?

Reply:
Will do!

Reply:
I posted some patches a while ago to start unifying things so that there's a single canonical version that gets copied and pasted, but those have been stalled for a while on reviewers, so for now we're mostly stuck with the ad-hoc way of doing things.

Reply:
> I posted some patches a while ago to start unifying things so that there's a single canonical version that gets copied and pasted, but those have been stalled for a while on reviewers, so for now we're mostly stuck with the ad-hoc way of doing things.

Can you point me to those pull requests.
BTW, I see that the compiler-rt change is in place. Do you find any issues there? I think the ordering with respect to libgcc is also okay. 
https://github.com/llvm/llvm-project/pull/107964/commits/b68bcc1415151bd84b5868aa2c98663069f45469#diff-f0c447f4acaa87cede210a02796f8a3b2f08f6d8acaf951ed4c65465bf37c967

Reply:
Oh, looks like I missed it. Sorry about that!

There's https://github.com/llvm/llvm-project/pull/97856, https://github.com/llvm/llvm-project/pull/97861, and https://github.com/llvm/llvm-project/pull/97872 (still a draft).

I want to try and land some test infrastructure first (https://github.com/llvm/llvm-project/pull/101927) so I can ensure the refactorings don't break anything though. There are some more patches that I need to finish doing, but haven't gotten to the rest yet given the current ones are stalled.